snowplow-tracker/src/main/java/com/snowplowanalytics/core/tracker/CrossDeviceParameter.kt

```diff
@@ -0,0 +1,29 @@
+package com.snowplowanalytics.core.tracker
+
+import com.snowplowanalytics.snowplow.controller.SessionController
+import com.snowplowanalytics.snowplow.controller.TrackerController
+
+/**
+ * The optional parameters to include in the query string added by [TrackerController.decorateLink]
+ */
+enum class CrossDeviceParameter {
+    /**
+     * Value of [SessionController.sessionId]
+     */
+    SESSION_ID,
+
+    /**
+     * Value of [Tracker.appId]
+     */
+    SOURCE_ID,
```

Comment:
Based on the JS tracker implementation, the `userId` should be before the `sourceId`.